### ISSUE TITLE
PDF fails veraPDF accessibility validation according to UA-2 standard

### ISSUE THREAD

**Issue by isaactpetersen** [wjschne/apaquarto#157]
Thank you very much for your development of this excellent package. I am looking to make my manuscripts more accessible. Quarto recently released accessibility checking of PDFs: https://quarto.org/docs/blog/posts/2026-03-05-pdf-accessibility-and-standards/.

I specify the accessibility checking using the following code:

```yaml
  apaquarto-pdf:
    pdf-standard: ua-2
```

When I implement the accessibility validation for a basic PDF generated from apaquarto, the PDF fails the accessibility validation with the following language:

```
[verapdf]: Validating index.pdf against PDF/UA2...FAILED

WARN: PDF validation failed for ua-2:
All destinations whose target lies within the current document shall be structure destinations
A link annotation and its associated content shall be enclosed in either a Link or Reference structure element
The logical structure of the conforming file shall be described by a structure hierarchy rooted in the StructTreeRoot entry of the document catalog dictionary, as described in ISO 32000-2:2020, 14.7
Every page that includes an annotation shall contain a Tabs entry in its page dictionary in accordance with ISO 32000-2:2020, Table 31, and its value shall be A, W or S
The default natural language for content and text strings shall be specified using the Lang entry, with a non-empty value, in the catalog dictionary
```

In case it's helpful to see the code for a reproducible example, the code for the manuscript is [here](https://github.com/DevPsyLab/QuartoManuscript). The GitHub Action run that shows the failing PDF validation (in section "Render Quarto Project") is [here](https://github.com/DevPsyLab/QuartoManuscript/actions/runs/24889163437/job/72876654235#step:11:425).

**Comment by wjschne** [comment]
@isaactpetersen, I believe this works now. Unfortunately, flextable and apa7 tables cannot be used with ua-2. This is [not flextable's fault](https://davidgohel.github.io/flextable/reference/knit_print.flextable.html#pdf-accessibility-pdf-ua-). When the underlying latex packages are updated, everything should work.

**Comment by isaactpetersen** [comment]
Thanks for looking at this! A few separate things:

1) On my [minimal working example](https://github.com/isaactpetersen/apaquartoReprex), the PDF accessibility validation now passes. Great! However, I do get the following warning (the link to the warning in the output is [here](https://github.com/isaactpetersen/apaquartoReprex/actions/runs/35028587234/job/104581475385#step:10:102)):

```
WARN: PDF accessibility: Neither unicode-math nor lua-unicode-math has been
```

Should we perhaps load either the `unicode-math` or `lua-unicode-math` package in the latex preamble (or do something else)?

2) On an actual Quarto manuscript I'm working on (not a minimal example), I get the following (the link to the repo is [here](https://github.com/DevPsyLab/studying-psychopathology-development-lifespan); the link to the output is [here](https://github.com/DevPsyLab/studying-psychopathology-development-lifespan/actions/runs/35025660203/job/104571941246#step:10:123)):

```
  [verapdf]: Validating index.pdf against PDF/UA2...FAILED
  
  WARN: PDF validation failed for ua-2:
  Content that is not considered real shall be an artifact
  A file in conformance with PDF/UA-2 shall not contain a reference to the .notdef glyph from any of the text showing operators, regardless of text rendering mode, in any content stream
```

I'm not sure whether this is an `apaquarto` issue or whether it's something I need to do differently. Any thoughts would be greatly appreciated. Thanks!
